docs/build-on-linea/tooling/analytics/mobula.md

```diff
@@ -0,0 +1,49 @@
+
+Seamless analysis of blockchain data is facilitated by [Mobula's](https://mobula.io) Analytics:
+
+### Why Mobula?
+
+- **Data Aggregation and Analytics**
+
+Aggregating vast datasets to provide a rich, unified view of information, the analytics engine of [Mobula](https://mobula.io) allows for in-depth analysis, unveiling critical patterns, trends, and insights. This powerful combination aids stakeholders in making informed decisions through comprehensive data analysis.
+
+- **User Experience and Functionality**
+
+Designed with user experience at the forefront, the platform features a straightforward interface that eases the complexity of data operations. Engineered to serve both novices and experts alike, [Mobula](https://mobula.io) ensures that its advanced data processing functionalities are within reach for all users. This approach significantly lowers the learning curve, facilitating more effective data utilization.
+
+- **Strategic Insight Generation**
+
+By transforming complex datasets into clear, actionable insights, this platform aids in streamlining decision-making across various contexts. It empowers users to uncover opportunities and tackle challenges, steering operational improvements and driving innovation strategies. With its broad applicability, [Mobula](https://mobula.io) becomes an indispensable tool for organizations striving to leverage data for strategic advancement comprehensively.
+
+&nbsp;
+
+<div class="center-container">
+  <div class="img-large">
+    <img
+      src="https://metacore.mobula.io/Capture%20d%E2%80%99e%CC%81cran%202024-04-03%20a%CC%80%2013.36.18.png"
+      alt="Mobula Analytics"
+    />
+  </div>
+</div>
+
+### Get started
+
+Looking to transform your data into actionable insights? Discover the power of [Mobula's](https://mobula.io) comprehensive analytics.
+
+Create a room with Mobula core team [Here](https://t.me/MobulaPartnerBot?start=Linea_Docs_Analytics)
+
+**More:**
+
+- **[Dive into our Documentation](https://docs.mobula.io)** - Delve into the extensive resources of [Mobula’s](https://docs.mobula.io) documentation. From detailed API guidelines to all-encompassing tutorials, our documentation is designed to empower users across all levels. Whether you're looking to integrate [Mobula's](https://docs.mobula.io) features into your application or eager to explore the full scope of functionalities we offer, you'll find everything required to unlock the full potential of your data-driven initiatives.
+
+- **[Explore Mobula App](https://mobula.io)** - Access the heart of cryptocurrency analytics with the [Mobula](https://mobula.io) app. Designed for ease of integration and enriched with powerful data insights, our app provides the tools you need to navigate the crypto market confidently. Get started today and harness the potential of advanced analytics tailored for your success.
```

Comment:
```suggestion
- **[Explore the Mobula app](https://mobula.io)** - Access the heart of cryptocurrency analytics with the [Mobula](https://mobula.io) app. Designed for ease of integration and enriched with powerful data insights, our app provides the tools you need to navigate the crypto market confidently. Get started today and harness the potential of advanced analytics tailored for your success.
```

Again, last sentence here is too marketing-oriented for the docs page. Either remove or soften please